core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -94,6 +99,14 @@ public void initializeLogging(LogConfig config) {
             configureSyslogHandler(config.syslog, errorManager, filterElements, handlers);
         }
 
+        final ErrorManager finalErrorManager = errorManager;
```

Comment:
Why is having this `finalErrorManager` necessary?

Reply:
Because it is captured in a lambda.  Or I mean... it used to be... oops :)

Reply:
Fixed.